users/tests.py

```diff
@@ -0,0 +1,62 @@
+# from django.test import TestCase
+# from asyncio import FastChildWatcher
+import pytest
+from django.core.exceptions import ValidationError
+from . import models
+
+
+PHONE_NUM = "0528305110"
+EMAIL = "yael@gmail.com"
+PASSWORD = "password"
+
+
+@pytest.fixture
+def user0():
+    return models.User(email=EMAIL, phone_number=PHONE_NUM, password=PASSWORD)
+
+
+def test_new_user(user0):
+    assert user0.email == EMAIL
+    assert user0.phone_number == PHONE_NUM
+    assert user0.name == 'User'
+    assert user0.profile_pic == 'profile_pic.svg'
+    assert user0.password == PASSWORD
+
+
+@pytest.mark.django_db
+def test_failed_creation_user1():
```

Comment:
please make the test name more descriptive
```suggestion
def test_invalid_phone_number():
```

Reply:
Done

Reply:
Same answer as line 38.